src/Symfony/Component/VarDumper/Caster/ConstStub.php

```diff
@@ -30,4 +30,27 @@ public function __toString()
     {
         return (string) $this->value;
     }
+
+    /**
+     * Creates ConstStub from bitfield flag.
+     *
+     * @param int    $value  The value of bitfield flag
+     * @param string $prefix The prefix filter applied on constant names
+     *
+     * @return self
+     */
+    public static function fromFlag($value, $prefix)
```

Comment:
i think the idea is to specify `$module` over `$prefix`.. then again im not sure it helps; i wont solve your issue, and for mine; i think passing the list explicitly (or derive it from `get_defined_constants()['Something']`), i.e. `$known` is the better approach.

Also an issue with prefix; i can specify multiple or so. What if i have simply `$some->field = Some::A | Some::B`?